core/example/solvers/main.cpp

```diff
@@ -0,0 +1,158 @@
+#include "definitions.h"
+#include "particles.h"
+//#include "ewald.h"
+#include "direct.h"
+#include "pme.h"
+//#include "mesh.h"
+
+#include <iomanip>
+#ifdef TDS_BENCHMARKING
+#include <fstream>
+#endif
+
+#define CRYSTAL_SIZE 2
+
+int main(int argc, char** argv)
+{
+  // Initialize the kokkos runtime.
+  Kokkos::initialize( argc, argv );
+
+
+#ifdef TDS_BENCHMARKING
+  std::ofstream directfile;
+  std::ofstream ewaldfile;
+  std::ofstream pmefile;
+#endif
+
+  // crystal size
+  int c_size = (argc >= 2)?atoi(argv[1]):CRYSTAL_SIZE;
```

Comment:
where is `atoi` defined? is this in the `std` namespace?

Reply:
I believe it is in `<cstdlib>`

Reply:
that header should be included then